src/token/bearer.rs

```diff
@@ -15,14 +14,14 @@ pub struct Bearer<L: Lifetime> {
 }
 
 impl<L: Lifetime> Token<L> for Bearer<L> {
-    fn access_token(&self) -> &str { &self.access_token }
-    fn scope(&self) -> Option<&str> { self.scope.as_ref().map(|s| &s[..]) }
-    fn lifetime(&self) -> &L { &self.lifetime }
-}
-
-impl<'a, L: Lifetime> Into<header::Authorization<header::Bearer>> for &'a Bearer<L> {
```

Comment:
Is there no new alternative here? How should users send the header?

Reply:
https://docs.rs/reqwest/0.9.2/reqwest/struct.RequestBuilder.html#method.header

It's complicated, There is two values now

Reply:
We can return https://docs.rs/reqwest/0.9.2/reqwest/header/struct.HeaderMap.html
What do you think about this solution?

Reply:
There's actually a dedicated method: https://docs.rs/reqwest/0.9.2/reqwest/struct.RequestBuilder.html#method.bearer_auth

So I think the user can just pass `bearer.access_token()`.

Reply:
Well seen!
I've impl this 5627530f9caf94d9f8f32bdc75d0ca5dc2d1247d